examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino

```diff
@@ -0,0 +1,59 @@
+/*
+ * This example check if the firmware loaded on the WiFi1010
```

Comment:
I suggest we rename the sketch to `CheckFirmwareVersion`.

As MKR WiFi 1010, MKR Vidor 4000 (and later Uno WiFi Rev2) are supported boards, I suggest you replace `WiFi1010` above with NINA module.

Reply:
ok i'll change in all the point required.